docs/reference/commandline/dockerd.md

```diff
@@ -1170,68 +1151,75 @@ This is a full example of the allowed configuration options on Windows:
 }
 ```
 
-The `default-runtime` option is by default unset, in which case dockerd will auto-detect the runtime. This detection is currently based on if the `containerd` flag is set.
+The `default-runtime` option is by default unset, in which case dockerd automatically detects the runtime.
+This detection is based on if the `containerd` flag is set.
 
 Accepted values:
 
 - `com.docker.hcsshim.v1` - This is the built-in runtime that Docker has used since Windows supported was first added and uses the v1 HCS API's in Windows.
 - `io.containerd.runhcs.v1` - This is uses the containerd `runhcs` shim to run the container and uses the v2 HCS API's in Windows.
 
 #### Feature options
-The optional field `features` in `daemon.json` allows users to enable or disable specific
-daemon features. For example, `{"features":{"buildkit": true}}` enables `buildkit` as the
-default docker image builder.
+
+The optional field `features` in `daemon.json` lets you enable or disable specific
+daemon features.
+
+```json
+{
+  "features": {
+    // enable or disable features
+  }
```

Comment:
Perhaps we could use a full example, or a "dummy" example to illustrate the expected format. Not sure what's best to illustrate this (the "boolean" was used with the possibility to _disable_ features that are enabled by default, but not sure if we currently have any);

```json
{
  "features": {
    "some-feature": true,
    "some-disabled-feature-thats-enabled-by-default": false
  }
}
```